CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Changelog
 - **Fix**: Fix `@Provides` graph parameters not getting passed on to extended child graphs.
 - **Fix**: Fix qualifiers on bindings not getting seen by extended child graphs.
 - **Fix**: Fix qualifiers getting ignored on accessors from `@Includes` dependencies.
+- **Fix**: Fix `@ContributesGraphExtension` missing binding receiver info for contributed bindings.
```

Comment:
Can omit this one since CGE's aren't released yet

```suggestion
```

Reply:
:+1: I was debating if it made sense or not, will remove